Flea/Pages/Booking.razor

```diff
@@ -1,6 +1,98 @@
 @page "/Booking"
-<h1>Booking</h1>
 
-@code {
+<div>
+    <input class="search-input" type="search" value="" placeholder="Søg efter booking" @oninput="Searchbar">
+    <button class="button res-new-booking" @onclick="NewReservation">Opret ny booking</button>
     
+    @foreach (var value in _shownReservation)
+    {
+        <div><ReservationOverview Name="@value.Name" Stande="@value.Stande" PaymentStatus="@value.PaymentStatus" OnDelete="@DeleteReservationInformation" OnEdit="@EditReservationInformation"/></div>
+    }
+</div>
+
+@code
+{
+    // Dummy data ---
+    class Reservation
+    {
+        public string Name { get; set; }
+        public int Stande { get; set; }
```

Comment:
Avoid using danish words in identifier names. (I am aware that it is merely placeholder but try keep it in mind still)

Reply:
Fixed